types/lib/common/benchmarkModelGenerator.d.ts

```diff
@@ -0,0 +1,94 @@
+export = BenchmarkModelGenerator;
+/**
+ * Generate a benchmark model.
+ *
+ * @class
+ */
+declare class BenchmarkModelGenerator {
+    /**
+     * Formats the given byte size into a human-readable string.
+     * @param {number} bytes The number of bytes.
+     * @param {number} [decimals=2] The number of decimal places to include.
+     * @returns {string} A formatted string representing the byte size.
```

Comment:
These are all public by default. Do you want them to be private or protected?

Reply:
Thanks! Addressed ✅